src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -85,19 +95,18 @@ const NavHeader = (props) => {
     ? false
     : !hasSearch || (hasSearch && coverPath?.length > 0);
   return (
-    <View>
-      <View style={{}}>
-        <StatusBar barStyle="dark-content" />
-        <View
-          style={[
-            styles.navContainer(isHeaderHide),
-            {
-              height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
-            }
-          ]}
-          imageStyle={{opacity: isHeaderHide ? 0 : 1}}>
-          {initialData?.coverImage === undefined && isLoading ? (
-            <Shimmer width={displayWidth} height={dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER} />
+    <Animated.View style={styles.container(animatedHeight)}>
+      <StatusBar barStyle="dark-content" />
+      <View
+        source={{uri: coverPath}}
+        style={[styles.navContainer(isHeaderHide, topPosition), heightWithCoverImage()]}
+        imageStyle={{opacity: isHeaderHide ? 0 : 1}}>
+        <Animated.View style={styles.headerImage(opacityHeaderAnimation)}>
+          <FastImage source={{uri: coverPath}} style={heightWithCoverImage()} />
+        </Animated.View>
+        <TouchableOpacity onPress={() => backScreen()} style={styles.backbutton}>
+          {coverPath && !isHeaderHide ? (
+            <MemoIcArrowBackCircle width={normalize(32)} height={normalize(32)} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a static height (`dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER`) to a dynamic one (`animatedHeight`) could lead to unexpected behavior if `animatedHeight` is not properly managed. Ensure that this value is always valid and within expected bounds.



<!-- This is an auto-generated comment by OSS CodeRabbit -->